pkg/version/version.go

```diff
@@ -109,6 +115,13 @@ func (i *Info) IsEqualExceptPatch(other *Info) bool {
 	return other.VdbMajor == i.VdbMajor && other.VdbMinor == i.VdbMinor
 }
 
+// IsLowerExceptPatch returns true if the receiver's major/minor version
+// is lower than the given version.
+func (i *Info) IsLowerExceptPatch(other *Info) bool {
```

Comment:
Prefer to use newer/older terms like other function. Can you rename this?
```suggestion
func (i *Info) IsOlderExceptPatch(other *Info) bool {
```